collect_app/src/main/java/org/odk/collect/android/widgets/AnnotateWidget.java

```diff
@@ -65,6 +65,9 @@ public AnnotateWidget(Context context, QuestionDetails prompt) {
         imageCaptureHandler = new ImageCaptureHandler();
         setUpLayout();
         addCurrentImageToLayout();
+        if (errorLoadingImage) {
```

Comment:
I think you need this at all... what about a check like?
```
        if (binaryName == null || imageView == null || imageView.getVisibility() == GONE) {
            annotateButton.setEnabled(false);
        }
```

there is a similar check:
```
if (binaryName == null) {
            annotateButton.setEnabled(false);
        }
```

in `setUpLayout()` method so you could remove it there.

Reply:
Done.